components/molecule/photoUploader/src/styles/settings.scss

```diff
@@ -43,6 +43,7 @@ $jc-photo-uploader: center !default;
 $p-photo-uploader-action-buttons: $p-m 0 !default;
 $p-photo-uploader-desktop: $p-xl !default;
 $p-photo-uploader: $p-m !default;
+$p-photo-uploader-content: $p-m !default;
```

Comment:
not necessary, we must offer an empty canvas and delegate the spacing margins-paddings L&F to the consumer of the space